IdentityCore/src/network/response_serializer/preprocessor/MSIDAADJsonResponsePreprocessor.m

```diff
@@ -58,12 +58,14 @@ - (id)responseObjectForResponse:(NSHTTPURLResponse *)httpResponse
     NSString *clientTelemetry = httpResponse.allHeaderFields[MSID_OAUTH2_CLIENT_TELEMETRY];
     if (![NSString msidIsStringNilOrBlank:clientTelemetry])
     {
+#if !EXCLUDE_FROM_MSALCPP
```

Comment:
Are you sure you don't want this information in CPP? This is a server response.

Reply:
MSAL CPP is not using these, and will parse the server response data in the cpp layer for all platforms.
Also, If I remove this MACRO, I have to add the definition of MSID_TELEMETRY_KEY_SPE_INFO in MSIDRequestTelemetryConstants.

Reply:
So, if you're not using this file, why don't you exclude entire file?

Reply:
Theoretically it is possible. But in order to exclude this file, I will have to exclude all the API calls from this class; since this file is excluded, why not exclude other similar files... this will be a chain reaction, I can't estimate how many works would be done... Excluding the telemetry is relatively easier...